apps/webservice/src/app/api/healthz/route.ts

```diff
@@ -1,5 +1,18 @@
 import { NextResponse } from "next/server";
+import IORedis from "ioredis";
+
+import { sql } from "@ctrlplane/db";
+import { db } from "@ctrlplane/db/client";
+
+import { env } from "~/env";
+
+export const GET = async () => {
+  try {
+    await new IORedis(env.REDIS_URL).ping();
```

Comment:
I don't think we want to set max retries to null for the health check (because apparently it waits forever), so just using the default which is 20